recipes/jkqtplotter/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "jkqtplotter" %}
+{% set version = "2019.11.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jkriege2/JKQtPlotter/archive/v{{ version }}.tar.gz
+  sha256: 94d39834c6f4e4afd4ad8511231a26d74fd88dc60d7b3ccd85b7ce00f32ff487
+
+build:
+  number: 0
+  skip: True  # [win and vc<14]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [unix]
+    - ninja   # [win]
+  host:
+    - qt
+    - {{ cdt('mesa-libgl-devel') }}      # [linux]
```

Comment:
`CDT`s should go under `build`.

Reply:
I moved it back to build in b25e30ebfc45bd0ea1c819a9cfe38705b4a5d927. I thought these provide the header files and libraries to link against on the host machine, so it would go into host?

Reply:
True. I'd like them to be in `host`, but for some reason that doesn't work.

Reply:
Ah, ok.